geocoding/lib/geocoding.dart

```diff
@@ -36,13 +35,28 @@ Future<List<Location>> locationFromAddress(
 /// When not supplied the currently active locale of the device will be used.
 /// The `localeIdentifier` should be formatted using the syntax:
 /// [languageCode]_[countryCode] (eg. en_US or nl_NL).
-Future<List<Placemark>> placemarkFromCoordinates(
+Future<List<Placemark>>? placemarkFromCoordinates(
   double latitude,
   double longitude, {
   String? localeIdentifier,
 }) =>
-    GeocodingPlatform.instance.placemarkFromCoordinates(
+    GeocodingPlatform.instance?.placemarkFromCoordinates(
```

Comment:
Enforce the `GeocodingPlatform.instance` not to be `null`.
```suggestion
    GeocodingPlatform.instance!.placemarkFromCoordinates(
```